api/proto/v1/common.proto

```diff
@@ -60,7 +60,8 @@ message InternalComputeInfo {
   string ip = 4;
   string mac = 5;
   string veth = 6;
-  Status status = 7;
+  string container_ip = 7;
+  Status status = 8;
```

Comment:
@zzengcs I will update this file, please remove this file from your commit.

Reply:
Fixed